app.py

```diff
@@ -1,53 +1,146 @@
 import streamlit as st
 import pandas as pd
-import time
-
-# def loader():
-#     latest_iteration = st.empty()
-#     bar = st.progress(0)
+import json
+import data_upload_DHIS2 as dhis2
 
 # Initializing variables
 form_types = ("ICCM", "OPD", "RHGynobs", "Vaccination")
+# Ideally, these could be set in this app?
+# dhis2_username = ''
+# dhis2_password = ''
+# DHIS2_Test_Server_URL = ''
 
-st.title("MSF OCR Tool")
 
-st.write("### File Upload ###")
+# Function definitions!
+# """ Creates a list of dataValues by going through each cell
+# and recording the column, header, and cell value of non-None
 
-# results = ocr.run()
+# :param df: an iterable of data frames
+# :return: a list of JSON-ready dataValues
+# """

```

Comment:
Whoohoo for docstrings!! Just move it to be after the function definition line (see my comment below).